examples/panel/stock-market-chatbot/requirements.txt

```diff
@@ -0,0 +1,16 @@
+panel==1.3.8 
+yfinance==0.2.37 
+hvplot 
+pandas 
+openai==1.12.0
+imagekitio
+holoviews==1.18.3
+duckdb==0.10.0
+bokeh==3.3.4
+python-dotenv
+selenium==4.18.1
+webdriver-manager==4.0.1
```

Comment:
do we need selenium? do a quick check and remove dependencies that we don't need

Reply:
resolved https://github.com/ploomber/doc/pull/124/commits/37a0f201a52161ab1b14b431bb8e6795d4caf9f5